pkg/datapath/linux/bigtcp/bigtcp.go

```diff
@@ -219,31 +219,31 @@ func InitBIGTCP(bigTCPConfig *Configuration) {
 		// modification and end up with an error, so best to be conservative
 		// and always reset all on error
 		modifiedDevices = append(modifiedDevices, device)
-		if haveIPv4 {
-			err = setGROGSOIPv4MaxSize(device,
-				bigTCPConfig.groIPv4MaxSize, bigTCPConfig.gsoIPv4MaxSize)
+		if haveIPv6 {
+			err = setGROGSOIPv6MaxSize(device,
+				bigTCPConfig.groIPv6MaxSize, bigTCPConfig.gsoIPv6MaxSize)
 			if err != nil {
-				log.WithError(err).WithField("device", device).Warnf("Could not modify IPv4 gro_max_size and gso_max_size%s",
+				log.WithError(err).WithField("device", device).Warnf("Could not modify IPv6 gro_max_size and gso_max_size%s",
 					disableMsg)
 				option.Config.EnableIPv4BIGTCP = false
 				option.Config.EnableIPv6BIGTCP = false
 				break
 			}
-			log.WithField("device", device).Infof("Setting IPv4 gso_max_size to %d and gro_max_size to %d",
-				bigTCPConfig.gsoIPv4MaxSize, bigTCPConfig.groIPv4MaxSize)
+			log.WithField("device", device).Infof("Setting IPv6 gso_max_size to %d and gro_max_size to %d",
+				bigTCPConfig.gsoIPv6MaxSize, bigTCPConfig.groIPv6MaxSize)
 		}
-		if haveIPv6 {
-			err = setGROGSOIPv6MaxSize(device,
-				bigTCPConfig.groIPv6MaxSize, bigTCPConfig.gsoIPv6MaxSize)
+		if haveIPv4 {
+			err = setGROGSOIPv4MaxSize(device,
+				bigTCPConfig.groIPv4MaxSize, bigTCPConfig.gsoIPv4MaxSize)
 			if err != nil {
-				log.WithError(err).WithField("device", device).Warnf("Could not modify IPv6 gro_max_size and gso_max_size%s",
+				log.WithError(err).WithField("device", device).Warnf("Could not modify IPv4 gro_max_size and gso_max_size%s",
 					disableMsg)
 				option.Config.EnableIPv4BIGTCP = false
 				option.Config.EnableIPv6BIGTCP = false
 				break
 			}
-			log.WithField("device", device).Infof("Setting IPv6 gso_max_size to %d and gro_max_size to %d",
-				bigTCPConfig.gsoIPv6MaxSize, bigTCPConfig.groIPv6MaxSize)
+			log.WithField("device", device).Infof("Setting IPv4 gso_max_size to %d and gro_max_size to %d",
+				bigTCPConfig.gsoIPv4MaxSize, bigTCPConfig.groIPv4MaxSize)
 		}
 	}
 
```

Comment:
nit: should we also modify the IPv4/IPv6 order for below err handling?

Reply:
64KB will be OK. Just less code change to make things work.